.github/workflows/ci.yaml

```diff
@@ -928,11 +929,12 @@ jobs:
           path: pytest-*.txt
       - name: Upload coverage artifact
         if: needs.info.outputs.skip_coverage != 'true'
-        uses: actions/upload-artifact@v3.1.2
+        uses: actions/upload-artifact@v4.3.1
         with:
-          name: coverage-${{ matrix.python-version }}-mariadb-${{
+          name: coverage-${{ matrix.python-version }}-${{
             steps.pytest-partial.outputs.mariadb }}
```

Comment:
`mariadb` is already included in `outputs.mariadb`. The output is the version just with `:` replaced by `-`.
https://github.com/home-assistant/core/blob/f792690e27cc3cbd48ee26feac079c7e23426eaa/.github/workflows/ci.yaml#L52